### PR TITLE
Fix apollo

### PR DESCRIPTION
**Pull request by ch4r10t33r**
Upgraded Apollo packages

## Description
Several developers were facing issues and warnings because of a very old version of the Apollo package being used as a dependency.
-

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
       "version": "1.3.8",
```
Comment (arddluma) [inline package-lock.json:9]: Please update here too

**Review by arddluma — CHANGES_REQUESTED** [review]
Please update in package-lock.json too

**Comment by ch4r10t33r** [comment]
> Please update in package-lock.json too

Should that get auto-generated?

File: package-lock.json (hunk 1 above)
Comment (ch4r10t33r) [reply package-lock.json:9]: We have to fix this. Package-lock.json should be part of .gitignore. I've added it now.

**Comment by IAmKio** [comment]
@ch4r10t33r can i just check that we were able to install and run the examples? I can't see any automated tests in the build pipeline...

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # OS
 .DS_Store
+package-lock.json
```
Comment (IAmKio) [inline .gitignore]: Just checking that this was supposed to be here? package-lock.json needs to be committed: 
https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#description

Comment (IAmKio) [reply .gitignore]: Oh ignore that you removed it as i was typing it

**Review by IAmKio — APPROVED** [review]

**Comment by ch4r10t33r** [comment]
> > Please update in package-lock.json too
> 
> Shouldn't that get auto-generated?

This is done.,

**Review by arddluma — APPROVED** [review]

**Pull request merged**